src/main/java/io/confluent/examples/streams/microservices/domain/Schemas.java

```diff
@@ -1,20 +1,21 @@
 package io.confluent.examples.streams.microservices.domain;
 
-import static io.confluent.examples.streams.microservices.util.MicroserviceUtils.ProductTypeSerde;
-import static io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG;
-
 import io.confluent.examples.streams.avro.microservices.Customer;
 import io.confluent.examples.streams.avro.microservices.Order;
 import io.confluent.examples.streams.avro.microservices.OrderValidation;
 import io.confluent.examples.streams.avro.microservices.OrderValue;
 import io.confluent.examples.streams.avro.microservices.Payment;
 import io.confluent.examples.streams.avro.microservices.Product;
 import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
-import org.apache.kafka.common.serialization.Serde;
-import org.apache.kafka.common.serialization.Serdes;
+
+import static io.confluent.examples.streams.microservices.util.MicroserviceUtils.ProductTypeSerde;
+import static io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG;
```

Comment:
@cadonna Just realizing that this should not happen -- using wrong styleguide triggered auto-reordering. Reverting this part.